users

```diff
@@ -0,0 +1 @@
":13000,"bursaryRequested":false,"bursaryApproved":false,"expired":false,"_id":{"$oid":"638d3138641cbe1419ea7145"}}],"phoneNumber":"(123) 424-2423","phoneNumberAreaCode":"1","photograph":true,"pronouns":"Prefer Not to Say","scunt":true,"scuntTeam":0,"shirtSize":"M","summerLocation":true,"summerLocationOther":"","userType":"frosh","utorid":"yashyash"},{"_id":{"$oid":"638d3291641cbe1419ea717a"},"approved":true,"froshDataFields":{"requested":["froshGroup","froshGroupIcon","isRetreat","accountCreatedAt","signInDate","scuntTeam","emailLabel","email","firstName","lastName","preferredName","legalName","pronouns","pronounOther","birthDate","utorid","discipline","shirtSize","phoneNumberLabel","phoneNumberAreaCode","phoneNumber","instagram","emergencyContactName","emergencyContactRelationship","phoneNumberEmergencyLabel","emergencyContactNumberAreaCode","emergencyContactNumber","bursaryRequested","medicalInfo","medication","allergies","allergiesMore","photograph","accessibility","accommodation","accommodationContact","accommodationOther","infoLabel","scunt","summerLocation","summerLocationOther","moveToToronto","commuterProgram","commuterProgramInformation","commuterProgramOther","commuterProgramStop"],"approved":[],"_id":{"$oid":"638d3291641cbe1419ea7179"}},"froshGroupName":"","froshGroupIcon":"","subCom":"","scuntJudgeBribePoints":0,"scuntTeam":-1,"firstName":"Yash","lastName":"Yash","preferredName":"Yash","email":"yash@vardhan.com","hashedPassword":"$2b$10$ojQTq3fz5ujwf0lUBf6QvO8Wvs3PMY4C8mJm.TQ5T1HSd60Z5JoRW","authScopes":{"requested":[],"approved":["faq:delete","faq:edit","announcements:create","announcements:edit","announcements:delete","accounts:read","accounts:edit","accounts:delete","email:send","timeline:create","timeline:edit","timeline:delete","signInFrosh:qr-code registration","scunt:judge missions","scunt:judge bribe points","scunt:exec refill bribe points","scunt:exec game controls","scunt:exec negative points","scunt:exec show missions","scunt:exec hide missions","scunt:exec create missions","scunt:exec delete missions","scunt:exec shuffle teams","scunt:exec view transactions","scunt:exec delete transaction","scunt:exec rename scunt teams","froshGroupData:all","froshGroupData:Alpha","froshGroupData:Beta","froshGroupData:Iota","froshGroupData:Phi","froshGroupData:Psi","froshGroupData:Rho","froshGroupData:Zeta","froshGroupData:Gamma","froshGroupData:Omega","froshGroupData:Chi","froshGroupData:Upsilon","froshGroupData:Pi","froshGroupData:Nu","froshGroupData:Delta","froshGroupData:Sigma","froshGroupData:Tau","froshGroupData:Kappa","froshGroupData:Theta","froshGroupData:Lambda","froshGroupData:Omicron","froshData:unRegisteredUsers"],"_id":{"$oid":"638d3291641cbe1419ea717b"}},"isDeleted":false,"accountCreatedAt":{"$date":"2022-12-04T23:39:37.788Z"},"lastUpdatedAt":{"$date":"2022-12-04T23:39:37.788Z"},"lastUpdatedFields":[],"canEmail":true,"isScuntDiscordLoggedIn":false,"userType":"leadur","completedAnnouncements":[],"__v":0,"scuntToken":"35674"}]
```

Comment:
Not sure what this file is doing here - we should get rid of this before merging the PR.

Reply:
Resolved in https://github.com/UofT-Frosh-Orientation/orientation-website/pull/573/commits/39194d4cfb1f2fecf14c82e5d4eaee1ff65175aa